lib/tests/common/mock_connector.hpp

```diff
@@ -55,7 +59,8 @@ static Configuration::Agent AGENT_CONFIGURATION { MODULES,
                                                   15,    // ping interval
                                                   30,    // task download connection timeout
                                                   120,   // task download timeout
-                                                  64 * 1024 * 1024 };  // default max-message-size
+                                                  64 * 1024 * 1024, // default max-message-size
```

Comment:
```suggestion
                                                  64 * 1024 * 1024,  // default max-message-size
```

Just to make the linter happy. 🙃